content/200-concepts/100-components/01-prisma-schema/06-relations/425-referential-actions/index.mdx

```diff
@@ -7,7 +7,7 @@ tocDepth: 2
 
 <TopBlock>
 
-Referential actions determine what happens to a record when a related record is deleted or updated. In [2.26.0](https://github.com/prisma/prisma/releases/tag/2.26.0) and later, you can define referential actions on your relation fields in your Prisma schema.
+Referential actions determine what happens to a record when a related record is deleted or updated. Since version [3.0.1](https://github.com/prisma/prisma/releases/tag/3.0.10) (and between [2.26.0](https://github.com/prisma/prisma/releases/tag/2.26.0) and 3.0.1 when using the preview feature flag `referentialActions`), you can define referential actions on your relation fields in your Prisma schema.
```

Comment:
```suggestion
Referential actions determine what happens to a record when a related record is deleted or updated. Since version 3.0.1 (and in version 2.26.0 or later when using the preview feature flag referentialActions), you can define referential actions on your relation fields in your Prisma schema.
```

Reply:
ping @tanberry

Reply:
I felt it was clear that since 3.0.1 provided full support, you only needed the flag up til then, but I changed it to hopefully make it crystal clear. How's this wording look? (And I added the links back; I think I just forgot to do it the first time).

Reply:
Yes that works for me.